src/test/java/guitests/MenuControlTest.java

```diff
@@ -117,6 +117,9 @@ public void menuControlTest() {
 
         // Testing First Panel selected
         assertEquals(0, (int) panelControl.getCurrentlySelectedPanel().get());
+        press(KeyCode.CONTROL).press(KeyCode.DIGIT1).release(KeyCode.DIGIT1).release(KeyCode.CONTROL);
+        assertEquals(0, (int) panelControl.getCurrentlySelectedPanel().get());
+        // TODO (currently not possible) test that panelsScrollPane.getHValue() is 0
```

Comment:
A public method in ScrollableListView wouldn't be breaking encapsulation. Is that what you're referring to?

Alternatively, what if we assert that the first issue of that panel is selected? After ensuring that it isn't selected before pressing Ctrl + 1.